resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -500,6 +500,7 @@ package object config extends Logging {
         " The driver can slow down and fail to respond to executor heartbeats in time." +
         " If enabling this flag, make sure your DNS server has enough capacity" +
         " for the workload.")
+      .internal()
```

Comment:
This makes the flag non-public. From the [code comment](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala#L38):

>   @param isPublic if this configuration is public to the user. If it's `false`, this
configuration is only used internally and we should not expose it to users.

I think we want to redesign the DNS lookup code path so that this flag is not needed in the future. So keeping the flag internal allow us to remove it later without worrying if there are some users depend on it.